cmd/antrea-agent/agent.go

```diff
@@ -185,9 +185,9 @@ func run(o *Options) error {
 		case v4Enabled && v6Enabled:
 			proxier = proxy.NewDualStackProxier(nodeConfig.Name, informerFactory, ofClient)
 		case v4Enabled:
-			proxier = proxy.NewProxier(nodeConfig.Name, informerFactory, ofClient, false)
+			proxier = proxy.NewProxier(nodeConfig.Name, informerFactory, ofClient, false, true)
```

Comment:
We need to use feature gate to determine if we need to enable endpointslice support in the proxy.

Reply:
Done.